src/bin/flamegraph.rs

```diff
@@ -177,10 +191,10 @@ struct Opt {
     /// Adds an outline to every frame
     #[clap(
         long = "stroke-color",
-        default_value = defaults::STROKE_COLOR,
+        default_value = defaults::UI_COLOR,
```

Comment:
This ends up changing the default from `none` to `#000000`. Was that intentional?